src/payment/create-payment-strategy-registry.ts

```diff
@@ -508,10 +508,12 @@ export default function createPaymentStrategyRegistry(
     );
 
     registry.register(PaymentStrategyType.BOLT, () =>
-        new BoltPaymentStrategy(
```

Comment:
Is the previous `BoltPaymentStrategy` still in use?

Reply:
It will not be used in the payment step, it will only be used by the bolt's app.
I don't know if that strategy will still be able to be used even if it's not in the registry, is that possible @capsula4 ?

Reply:
if it's not in the registry most likely is not in use, since the new strategy doesn't really extend the old one. You can do a search in the repo but most likely is not in use.